cl/search/api_utils.py

```diff
@@ -27,10 +30,24 @@ def get_object_list(request, cd, paginator):
         group = True
 
     total_query_results = 0
-    if cd["type"] == SEARCH_TYPES.ORAL_ARGUMENT and waffle.flag_is_active(
-        request, "oa-es-active"
-    ):
-        search_query = AudioDocument.search()
+
+    is_oral_argument_active = cd[
+        "type"
+    ] == SEARCH_TYPES.ORAL_ARGUMENT and waffle.flag_is_active(
+        request, "oa-es-activate"
+    )
+    is_people_active = cd[
+        "type"
+    ] == SEARCH_TYPES.PEOPLE and waffle.flag_is_active(
+        request, "p-es-activate"
+    )
+
+    if is_oral_argument_active or is_people_active:
+        search_query = (
+            AudioDocument.search()
+            if is_oral_argument_active
+            else PersonDocument.search()
+        )
```

Comment:
Call me crazy, but isn't it a lot easier to just do:

```suggestion
    if is_oral_argument_active:
        search_query = AudioDocument.search()
    elif is_people_active:
        search_query = PersonDocument.search()
```

Removes two lines, works the same, and is easier to read?

Reply:
Sure, changed it.